### ISSUE TITLE
can offer  the network  structure of second from tensorboard ?

### ISSUE THREAD

**Issue by zyxcambridge** [open-mmlab/mmdetection3d#55]
pytorch 1.4 can support tensorboard ?
tensorboard can show the network structure ?
i want to get the network structure of second or pointpillars ,
can you help me ?

**Comment by ZwwWayne** [comment]
We only use tensorboard to record the loss, learning rate, and evaluation results. To know the structures of models, you can check the log. We already log the structure of model in the log by [this line of code](https://github.com/open-mmlab/mmdetection3d/blob/master/tools/train.py#L140).
